### PR TITLE
Examples: Verify Webapp

### PR DESCRIPTION
**Pull request by erin-at-work**
### _Summary_

- Adds `examples` directory
- Adds 'Verify Webapp` sample app

image

### CODE REVIEW

**Comment by cb-jake** [comment]
Rando question but why are the app directories prefixed with `a-`?

File: examples/verify-webapp/a-wallet/src/main.ts [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  }
+}
+
+function isAndroid() {
+  const isAndroidUserAgent = /(android)/i.test(navigator.userAgent);
+  return isAndroidUserAgent;
+}
+
+async function signEthereum() {
+  console.log("signEthereum");
```
Comment (cb-jake) [inline examples/verify-webapp/a-wallet/src/main.ts:27]: is this log needed?

File: examples/verify-webapp/a-wallet/src/main.ts [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+      app.innerHTML = `${err.message}. Redirecting back ...`;
+    }
+
+    // Go back to app
+    setTimeout(() => window.history.back(), 800);
+  }
+}
+
+async function handleEthereum() {
+  console.log("handleEthereum");
```
[the remark this answers is not in the document]
Comment (erin-at-work) [reply examples/verify-webapp/a-wallet/src/main.ts:65]: I left logs in as part of debugging the test app

File: examples/verify-webapp/a-profile-app/src/App.tsx [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+        fetchImages(address);
+      } else {
+        // Error state
+        console.error("Error verifying user");
+      }
+    }
+  }, [address]);
+
+  const handleOnClick = async () => {
+    console.log("clicked");
```
Comment (cb-jake) [inline examples/verify-webapp/a-profile-app/src/App.tsx:73]: rouge log?

File: examples/verify-webapp/a-profile-app/src/App.tsx [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+    window.location.href = VERIFY_URL;
+  };
+
+  const handleOnSelect = (img: string) => {
+    console.log("selected");
```
Comment (cb-jake) [inline examples/verify-webapp/a-profile-app/src/App.tsx:78]: rouge log?

File: examples/verify-webapp/a-profile-app/src/App.tsx [hunk 5]
```diff
@@ -0,0 +1,180 @@
...
+        Boolean(img.image_preview_url || img.collection.image_url),
+      );
+
+      setImages(validImgUrls);
+    } catch (err) {
+      console.error(err);
+    }
+  }
+
+  console.log(address);
```
Comment (cb-jake) [inline examples/verify-webapp/a-profile-app/src/App.tsx:55]: rouge log?

**Review by cb-jake — COMMENTED** [review]
lgtm just a few clean up items on the left over console logs

**Pull request merged**